user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md

```diff
@@ -0,0 +1,136 @@
+
+1. Right-click the Visual page and select *Set as active \<element name> element Visio file* > *New blank*.
+
+1. In the dialog box asking whether you want to assign a new drawing, click *Yes*.
+
+At this point, your Microsoft Visio application should start up automatically, and you can start modifying your new Visio file.
+
+## Step 2: Set up an element alarm color LED shape
+
+1. Remove all shapes from your Visio page by pressing CTRL+A and then Delete.
+
+1. In the ribbon at the top, go to the *Tools* section of the *Home* tab, and select the ellipse.
+
+1. Draw a circle on your page by clicking and dragging the mouse on it while this tool is selected.
+
+   ![Add ellipse to Visio page](~/user-guide/images/AddEllipseToVisioPage.gif)
+
+1. Right-click the newly created ellipse and select *Data* > *Define Shape Data*.
+
+1. Set the *Label* field to `Element`.
+
+1. Set the *Value* field to the name of your element.
+
+1. Click *OK*.
+
+1. Save your Visio file.
+
+In DataMiner Cube, your Visual Overview should now be updated with your changes. You should see an interactive ellipse shape that is filled with the current alarm state color.
+
+![Basic element shape](~/user-guide/images/BasicElementShape.webp)
+
+## Step 3: Show more information efficiently
+
+1. In the *Tools* section of the *Home* tab of the ribbon, select the rectangle.
+
+1. Draw a rectangle on your page by clicking and dragging the mouse on it while this tool is selected.
+
+1. Right-click the newly created rectangle and select *Data* > *Define Shape Data*.
+
+1. Set the *Label* field to `Parameter`.
+
+1. Set the *Value* field to `350`.
+
+   This is the parameter ID of the *Total Processor Load* parameter.
+
+1. Click *OK*.
+
+1. Double-click the rectangle shape and enter the shape text `*`.
+
+1. Right-click the ellipse and select *Shape Data* to open the *Shape Data* panel.
+
+1. In the *Shape Data* panel, set the value of the "Element" shape data to "*".
+
+1. In the *Tools* section of the *Home* tab of the ribbon, select *Pointer Tool*.
+
+1. Select both shapes on your page, right-click and select *Group* > *Group*.
+
+   ![Group shapes](~/user-guide/images/GroupShapes.gif)
+
+1. Right-click the newly created shape group and select *Data* > *Define Shape Data*.
+
+1. Add shape data with label `Element` and set its value to the name of your element.
+
+1. Save your Visio file.
+
+In DataMiner Cube, you should see two colored shapes, one of which displays the value of your parameter.
+
+![Group shape result](~/user-guide/images/GroupShapeResult.webp)
+
+## Step 4: Clean up the drawing
+
+1. Right-click the shape group and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Enabled` and value `false`.
+
+1. Right-click the shape group and select *Group* > *Open Group*.
+
+1. Right-click the ellipse shape and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Enabled` and value `true`.
```

Comment:
Could you explain what the purpose of this is? I think some users might feel a little lost without that info. Right now, you explain it all in the paragraph at the end, but it I think it would be better to give the info bit by bit whenever relevant.